build.sbt

```diff
@@ -1,24 +1,24 @@
 import sbt.Keys.organization
 
 val commonSettings: Seq[SettingsDefinition] = Seq(
-  scalaVersion := "2.12.10",
+  scalaVersion := "2.13.1",
 
   libraryDependencies := Seq(
     "ch.qos.logback" % "logback-classic" % "1.2.3",
-    "com.github.mpilquist" %% "simulacrum" % "0.10.0",
+    "org.typelevel" %% "simulacrum" % "1.0.0",
     "com.gu" %% "fezziwig" % "1.3",
     "com.gu" %% "ophan-event-model" % "0.0.17" excludeAll ExclusionRule(organization = "com.typesafe.play"),
     "com.gu" %% "acquisitions-value-calculator-client" % "2.0.5",
     "com.squareup.okhttp3" % "okhttp" % "3.9.0",
-    "com.typesafe.scala-logging" %% "scala-logging" % "3.7.2",
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2",
     "io.circe" %% "circe-core" % "0.12.1",
-    "org.scalatest" %% "scalatest" % "3.0.1" % "test",
-    "org.scalactic" %% "scalactic" % "3.0.1",
-    "org.typelevel" %% "cats-core" % "1.0.1",
+    "org.scalatest" %% "scalatest" % "3.1.1" % "test",
+    "org.scalactic" %% "scalactic" % "3.1.1",
+    "org.typelevel" %% "cats-core" % "2.1.1",
     "com.amazonaws" % "aws-java-sdk-kinesis" % "1.11.465",
-    "com.gu" %% "thrift-serializer" % "4.0.3",
-    compilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
+    "com.gu" %% "thrift-serializer" % "4.0.3"
   ),
+  scalacOptions += "-Ymacro-annotations",
```

Comment:
this is needed to get simulacrum working which uses the scala macros now in 2.13 instead of macro paradise